lib/applications/routes.js

```diff
@@ -4,53 +4,322 @@
  * @fileoverview Applications routing functions
  *
  */
+const config = require('config');
+const _ = require('lodash');
+const types = require('punchcard-content-types');
+const uuid = require('uuid');
+
+const database = require('../database');
+const model = require('./model');
+const utils = require('../utils');
 
 /**
  * All Webhooks Landing page
  * @param {object} req - HTTP Request
  * @param {object} res - HTTP Response
  * @param {object} next - Express callback
  *
+
+ * @returns {promise} promise wrapping database promise
  */
-const all = () => {};
+const all = (req, res, next) => {
+  return new Promise((resolve, reject) => {
+    return database
+      .select('*')
+      .from(`${config.applications.base}`)
```

Comment:
Our Database table names aren't configurable. This is [always going to be `applications`](https://github.com/punchcard-cms/punchcard/blob/master/lib/database/schemas.js#L238-L239). This is an incorrect use of configuration